modules/module/client-react/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@module/module-client",
```

Comment:
@evgenijolshanskij It does not include the full stack, because this information is redundant in the name of the module and will make imports longer. It is assumed that all the modules of the same application will use the same technology stack.

Reply:
@vlasenko Then it would be logical to rename `client-react` with `client`. It is no need to specify the stack as we know what are we working with.

Reply:
@vlasenko Actually, you are right, the module name must match the dirname. Because how else we will add `client-angular` subdir with Angular implementation..., I'll change that